RELEASE.md

```diff
@@ -0,0 +1,35 @@
+# Release Process
+
+Official Falcosecurity rules releases are automated using GitHub Actions. Each ruleset is released individually and each version is tied to a specific git tag.
+
+## Releasing a ruleset
+
+In this repo, each ruleset is a standalone YAML file in the `/rules` directory (e.g. `falco_rules.yaml`, `application_rules.yaml`, ...). Each ruleset is released and versioned individually. When we release a ruleset, we do the following process:
+
+1. Determine a new version for the given ruleset (see the [section below](#versioning-a-ruleset))
+2. Create a new Git tag with the name convention `*name*-rules-*version*` (e.g. `falco-rules-0.1.0`, `application-rules-0.1.0`, ...). The naming convention is required due to this repository being a [monorepo](https://en.wikipedia.org/wiki/Monorepo) and in order to be machine-readable.
+3. A GitHub action will validate the repository [registry](./registry.yaml) and release the new OCI artifact in the packages of this repository
+
+## Versioning a ruleset
+
+The version of the official Falco rulesets is compatible with [SemVer](https://semver.org/) and must be meaningful towards the changes in its structure and contents. To define a new version `x.y.z` for a given ruleset, consider the following guidelines. 
+
+**NOTE:** *The versioning guidelines also apply to any versioned ruleset not maintained in this repository (such as the ones in [falcosecurity/plugins](https://github.com/falcosecurity/plugins)), including the ones distributed by third parties. These are best practices that guarantee the correct behavior of Falco when updating a give ruleset to a new version.*
+
+- `z` is incremented when when you make backwards compatible bug fixes. In this case, the ruleset can be updated in a given Falco without needing to update Falco, its plugins, or its configuration. Examples:
+    - Adding one or more lists, macros, or rules
+    - Enabling at default one or more rules that used to be disabled
+    - Adding or removing items for one or more lists
+    - Adding or removing exceptions for one or more Falco rules (without needing a `required_engine_version`)
+    - Chaning the condition for one or more rules or macros by still preserving their logical security scope (e.g. making them less noisy)
+- `y` is incremented when you add functionality in a backwards compatible manner. In order to be accepted, the new ruleset mandates updating the version of Falco, changing its configuration, or updating/installing one or more plugins. Examples:
```

Comment:
```suggestion
- `y` _(minor number)_ is incremented when you add functionality in a backward-compatible manner. In order to be accepted, the new ruleset may mandates updating the version of Falco, changing its configuration, or updating/installing one or more plugins. Examples:
```

As per [semver spec](https://semver.org/#spec-item-7):
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. **It MAY be incremented if substantial new functionality or improvements are introduced within the private code**. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

Adding a new rule, without changing anything else in the rules, is a substantial new functionality, so we should increment the minor number. In this specific case, no new version of Falco or plugins may be needed.